mmc-utils/mmc-unmount.c

```diff
@@ -136,8 +136,6 @@ main (int argc, char *argv[])
   GFile *file;
   GMount *mount;
 
-  g_type_init ();
-
```

Comment:
I would have done it like:

#if !GLIB_CHECK_VERSION(2,35,0)
  g_type_init();
#endif

but not really a problem

Reply:
I saw this earlier (in MCE, if IIRC). But decided to leave it as is.
Do you want me to add this check?

Reply:
no, leave it as it is